workers/common/src/main/kotlin/common/OrtMappings.kt

```diff
@@ -209,18 +209,9 @@ fun OrtAnalyzerRun.mapToModel(analyzerJobId: Long) =
 
 fun OrtCopyrightFinding.mapToModel() = CopyrightFinding(statement = statement, location = location.mapToModel())
 
-fun OrtSnippetFinding.mapToModel() = SnippetFinding(
```

Comment:
My IDE has a structure overview which can show the members of a file in alphabetical order.